content/04-resources/01-faq.mdx

```diff
@@ -9,42 +9,37 @@ metaTitle: FAQ
   The software implements the official Ethereum Classic specification and Ethereum Classic Improvement Proposals (ECIPs) introduced by ongoing efforts and discussed across teams in the ecosystem.
 
 ### Why relaunch Mantis?
-
   * The goal of resurrecting Mantis is to make ETC stronger and to drive more independence and more decentralization in the ecosystem.
 
 ### Is Mantis a Wallet or a Node client?
-
   * Mantis is the [node client](#) and a [wallet](#). You can download them separately and use any according to your needs. These are delivered and released separately and each has its own installation and configuration options.
   The main differences between them is their usage. Node clients track the global chain state, while the wallet allows you to send and receive cryptocurrency transactions on chain. The client is a command line tool, and the wallet has a GUI interface.
 
 ### What are the benefits of Mantis?
-
   * Another client in the ETC space is a huge benefit to decentralization.
   * Written from scratch in Scala, specifically for ETC
   * IOHK backing and supporting the client in the long term
   * Additional features with Sagano, like OBFT, PT, etc...
 
 ### What language is Mantis written in?
-
   * Client: Scala + some solidity code and Nix here and there
   * Wallet: Typescript
 
 ### Does Mantis have fast-sync/archive-sync/pruning/API/monitoring/support?
 
   * Yes to all of the above.
 
-### Can I use mantis to connect to Sagano testnet?
+### Can I use Mantis to connect to Sagano testnet?
 
-  * ATM, Mantis is the only client that can connect to the Sagano testnet.
+  * At present, Mantis is the only client that can connect to the Sagano testnet.
 
 ### How fast does Mantis sync with Mainnet?
 
-  * Approx 12 hours to full mainnet Ethereum Classic sync with fastsync.
+  * The time to synchronize the mainnet can vary widely from case to case. Machine specs, and conditions mean a mainnet sync can take anywhere from 6 hours to 3 days.
```

Comment:
Can link to the system requirements page that has some specifics (eg. storage recommendations.)

Reply:
Ah there is not one system requirements page... might be helpful though (could also be part of "getting started"?) Or perhaps here.

Reply:
Done.